modtools/create-unit.lua

```diff
@@ -56,8 +56,12 @@ Creates a unit.  Usage::
 
     -name entityRawName
         Set the unit's name to be a random name appropriate for the
-        given entity. \\LOCAL can be specified instead to automatically
-        use the fort group entity in fortress mode only.
+            given entity. 
```

Comment:
Since you're editing this file, could you remove this entire string and use `dfhack.script_help()` instead where it is used? Help text has been moved to the docs directory. See https://github.com/DFHack/scripts/blob/12d78b25994521451392d8538c0d2db22499bfe3/quickfort.lua#L54 for an example

Reply:
Done [8dd1dd5](https://github.com/DFHack/scripts/pull/461/commits/8dd1dd5fffe3e348546faba6ec79caaa506edc6b)